tests/robot-tests/tests/admin_and_public/bau/prerelease_visibility.robot

```diff
@@ -111,27 +137,37 @@ Check scheduled release isn't visible on public Table Tool
 
 Go to public release URL and check release isn't visible
     user navigates to public frontend    ${PUBLIC_RELEASE_LINK}
+    user waits until page contains    Page not found
     user waits until page does not contain    ${PUBLICATION_NAME}
+
+Check methodology isn't visible on public Methodologies page
+    user navigates to methodologies page on public frontend
+    user waits until page contains    %{TEST_THEME_NAME}
+    user opens accordion section    %{TEST_THEME_NAME}
+    user checks page does not contain    ${PUBLICATION_NAME}
+
+Check methoodlogy isn't accessible via URL
+    user navigates to public frontend    ${PUBLIC_METHODOLOGY_LINK}
     user waits until page contains    Page not found
 
 Go to admin release summary
     user navigates to admin dashboard    Bau1
     user navigates to scheduled release page from dashboard    ${PUBLICATION_NAME}
     ...    ${RELEASE_NAME}
 
-Approve release for immediate publication but don't wait to finish
-    user clicks link    Sign off
-    user waits until h2 is visible    Sign off
-    user waits until page contains button    Edit release status
-    user clicks button    Edit release status
-    user waits until h2 is visible    Edit release status
-    user clicks radio    Approved for publication
-    user enters text into element    id:releaseStatusForm-internalReleaseNote    Approved by UI tests
-    user clicks radio    Immediately
-    user clicks button    Update status
-    user waits until h2 is visible    Sign off
-    user checks summary list contains    Current status    Approved
+Approve release for immediate publication
+    user approves original release for immediate publication
 
-Go to public release URL again and check release isn't visible
+Check release has been published
     user navigates to public frontend    ${PUBLIC_RELEASE_LINK}
-    user waits until page contains    Page not found
+    user waits until page contains title caption    Calendar year 2000
+    user waits until h1 is visible    ${PUBLICATION_NAME}
+
+Check methodology has been published
+    user navigates to methodologies page on public frontend
+    user waits until page contains    %{TEST_THEME_NAME}
+    user opens accordion section    %{TEST_THEME_NAME}
+    user clicks link    ${PUBLICATION_NAME}
+
+    user waits until page contains title caption    Methodology
+    user waits until h1 is visible    ${PUBLICATION_NAME}
```

Comment:
Would it make sense to check it's also visible on the public methodologies page as well after this?